File: src/main/java/org/intellij/sdk/language/angelscript/psi/impl/AngelScriptNamedElementImpl.java
package org.intellij.sdk.language.angelscript.psi.impl;

import com.intellij.extapi.psi.ASTWrapperPsiElement;
import com.intellij.lang.ASTNode;
import org.intellij.sdk.language.angelscript.psi.AngelScriptNamedElement;
import org.jetbrains.annotations.NotNull;

public abstract class AngelScriptNamedElementImpl extends ASTWrapperPsiElement implements AngelScriptNamedElement {
	public AngelScriptNamedElementImpl( @NotNull ASTNode node ) {
		super( node );
	}
}

File: src/main/kotlin/com/enderzombi102/css/lang/angelscript/psi/impl/AngelScriptNamedElementImpl.kt
package com.enderzombi102.css.lang.angelscript.psi.impl

import com.intellij.extapi.psi.ASTWrapperPsiElement
import com.intellij.lang.ASTNode
import com.enderzombi102.css.lang.angelscript.psi.AngelScriptNamedElement

abstract class AngelScriptNamedElementImpl(node: ASTNode) : ASTWrapperPsiElement(node), AngelScriptNamedElement